### PR TITLE
PPR/MHR dashboard updates

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#13672

Majority of issues in this ticket were already complete. 
Functionality to add an MHR parent and child via search for the child needs a separate ticket.
Charles will cover autoscroll and highlight styling for MHR registrations in his ticket [#13848].(https://github.com/bcgov/entity/issues/13848)
Rest of issues were verified to be complete

*Description of changes:*
-Updated text for dashboard search validation.
-Updated text for dashboard search tooltip.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by cameron-eyds — APPROVED** [review]
LGTM!

**Review by dimak1 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]

**Comment by dimak1** [comment]
@owillborn can you please sync this with `main` and bump up the version?

File: ppr-ui/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ppr-ui",
-  "version": "0.3.53",
+  "version": "0.3.54",
```
Comment (dimak1) [inline ppr-ui/package.json]: Thanks for the update. **package-lock.json** should have an updated version as well. Maybe forgot to run `npm install` ?

**Review by dimak1 — CHANGES_REQUESTED** [review]

**Comment by dimak1** [comment]
@owillborn just one comment about version update.

File: ppr-ui/package.json (hunk 1 above)
Comment (owillborn) [reply ppr-ui/package.json]: Done

**Review by dimak1 — APPROVED** [review]

**Pull request merged**